l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -81,6 +84,8 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
             l1LegacyBridge = _l1LegacyBridge;
             // l2StandardToken and l2TokenBeacon are already deployed on ERA, and stored in the proxy
         }
+
+        __Ownable_init(); // Shall we use 2 step version instead? 
```

Comment:
Another questions how to do better

Reply:
Same as before, we have to use the current version